maliput_ros_interfaces/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+##############################################################################
+# Project
+##############################################################################
+
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+project(maliput_ros_interfaces)
```

Comment:
nit: Simply nit to think/define naming.

If it is `maliput_ros_interfaces` then the next package that will integrate maliput with ros will be called: `maliput_ros`, right?

Should we rename the repo to be `maliput_ros_pkgs`?

Reply:
In my head, I have the following packages:

- maliput_ros_interfaces: message definition.
- maliput_ros_translation: message translation between ROS and maliput.
- maliput_ros: this one will have the query service manager, the node(s) and the launch configuration.

I think we can rename repo at any point, yes. I don't mind leaving it as ros2_maliput either.